### PR TITLE
Minor Changes to Eval Command

### PR DESCRIPTION
**Pull request by Shivansh-007**

### CODE REVIEW

File: bot/exts/utils/_eval_helper.py [hunk 1]
```diff
@@ -278,14 +279,14 @@ def format_hastebin_output(self, output: dict, result: str) -> Embed:
...
-        if result.count("\n") > 40:
+        if result.count("\n") > self.max_lines:
             result = [
                 f"{i:02d} | {line}" for i, line in enumerate(result.split("\n"), 1)
             ]
             result = result[: self.max_lines]  # Limiting to only 11 lines
             program_output = "\n".join(result) + "\n... (truncated - too many lines)"
 
-        elif len(result) > 1991:
+        elif len(result) > 1000:
```
Comment (gustavwilliam) [inline bot/exts/utils/_eval_helper.py:289]:
```suggestion
        elif len(result) > self.max_output_length:
```
Something like this should be used. We don't want hard coded values.

File: bot/exts/utils/eval.py [hunk 2]
```diff
@@ -167,19 +167,18 @@ async def eval_command(
                 except ValueError:
                     pass
 
-            if len(result) > 1991 or result.count("\n") > 40:
+            format_output = FormatOutput(language=lang)
+
+            if len(result) > 1000 or result.count("\n") > format_output.max_lines:
```
Comment (gustavwilliam) [inline bot/exts/utils/eval.py:172]: What's up with this `1000`? I don't quite get what the hardcoded values are supposed to represent.

**Review by gustavwilliam — CHANGES_REQUESTED** [review]
I don't understand what's up with these hardcoded vales.

Also, why are you changing things regarding the eval helper, like. removing the last few lines?

**Comment by Shivansh-007** [comment]
> I don't understand what's up with these hardcoded vales.
> 
> Also, why are you changing things regarding the eval helper, like. removing the last few lines?

Since those lines weren't needed. For example:
```py
a = 10
b = 15
if a > 5 or b > 10:
    # DO THIS
   return False
# and then you do.
if a > 3 or b > 5:
   if a> 5:
      # this would never be called
```
Hence i removed it

**Review by gustavwilliam — APPROVED** [review]
As a code review, it looks good.

**Pull request merged**